extensions/sql-migration/src/main.ts

```diff
@@ -76,16 +81,47 @@ class SQLMigration {
 	async launchMigrationWizard(): Promise<void> {
 		let activeConnection = await azdata.connection.getCurrentConnection();
 		let connectionId: string = '';
+		let serverName: string = '';
 		if (!activeConnection) {
 			const connection = await azdata.connection.openConnectionDialog();
 			if (connection) {
 				connectionId = connection.connectionId;
+				serverName = connection.options.server;
 			}
 		} else {
 			connectionId = activeConnection.connectionId;
+			serverName = activeConnection.serverName;
+		}
+		if (serverName) {
+			const api = (await vscode.extensions.getExtension(mssql.extension.name)?.activate()) as mssql.IExtension;
+			if (api) {
```

Comment:
If this was to ever happen that'd be a pretty catastrophic error in general so I'd say this check should be unnecessary.